packages/interface/src/errors.ts

```diff
@@ -65,3 +65,83 @@ export class InvalidCryptoTransmissionError extends Error {
 
   static readonly code = 'ERR_INVALID_CRYPTO_TRANSMISSION'
 }
+
+export enum messages {
```

Comment:
Maybe messages should stay in the libp2p package since they do not need to be standardized

Reply:
I think each module should probably have it's own specific messages, otherwise we'd need to ship libp2p to fix an autonat typo, for example.

Some generic messages might live in `@libp2p/interface`.